includes/admin/importers/class-wc-product-csv-importer-controller.php

```diff
@@ -128,7 +128,7 @@ public function get_next_step_link( $step = '' ) {
 
 		$params = array(
 			'step'            => $keys[ $step_index + 1 ],
-			'file'            => $this->file,
+			'file'            => str_replace( DIRECTORY_SEPARATOR, '/', $this->file),
```

Comment:
Needs a space after `$this->file` before the closing parentheses.

You can use phpcs to automatically check for format problems like quotes or spaces. 

From the woocommerce folder run:
```
vendor/bin/phpcs --standard=phpcs.ruleset.xml includes/admin/importers/class-wc-product-csv-importer-controller.php
```

or you can just do `grunt phpcs` but that will check every woocommerce file.